radix-engine-interface/src/blueprints/resource/access_rules.rs

```diff
@@ -69,6 +56,7 @@ impl MethodEntry {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd, ScryptoSbor, ManifestSbor)]
 pub enum MethodPermission {
     Public,
+    OuterObjectOnly,
```

Comment:
As discussed, we're going to rename `MethodPermission` to `MethodAccessibility`, understanding there is overhead to downstream clients. Let's also add some comments.